bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -137,7 +137,7 @@ func (r *KubeadmConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	// Look up the owner of this KubeConfig if there is one
 	configOwner, err := bsutil.GetConfigOwner(ctx, r.Client, config)
-	if apierrors.IsNotFound(err) {
+	if apierrors.IsNotFound(errors.Cause(err)) {
```

Comment:
@vincepri if I'm not wrong this is not necessary anymore given that https://github.com/kubernetes/apimachinery/commit/fb0f31795c1a4c82d7f141959c0605bb5c18977f#diff-9a385cdfb85037788a2ffd87c57d93ca16e7f1c98587eb362f6262a0f176bd2a merged in v1.19.2

Reply:
I don't know if that's actually enough, because errors.Wrap() from the pkg/errors library (not the stdlib one) wraps in a different way. I need to test it out, when we were working with @CecileRobertMichon yesterday, we saw this issue, although that might have been in v0.3.x

Reply:
Digging through the code, this seems to be fine for v0.4.x, although it definitely applies to v0.3.x codebase

@CecileRobertMichon @ncdc Do we want to have this PR target release-0.3 instead?

Reply:
SGTM